rt/transports/jms/src/main/java/org/apache/cxf/transport/jms/JMSConduit.java

```diff
@@ -510,6 +510,7 @@ private synchronized void shutdownListeners() {
         }
     }
     public synchronized void close() {
+        jmsConfig.resetCachedReplyDestination();
```

Comment:
I believe we should call it after the `shutdownListeners()` (because we still have listeners on the queue being deleted) but before `ResourceCloser.close(connection)`.

Reply:
I moved it.